### PR TITLE
Fixed Issue #6

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
The issue assigned to me #6 has been resolved !
I've used Status codes 200,201,202 and 400.
The Successful status codes give the user a clear idea of what's going on
The 4xx status codes havnt been used explicitly (only 400 is used) as it could lead to detailed error which could compromise the security in return !

### CODE REVIEW

File: resume/routes.py [hunk 1]
```diff
@@ -14,49 +14,49 @@
...
             next_page = request.args.get('next')            
             if next_page:
-                return redirect(next_page)
+                return redirect(next_page),200
             else:
-                return redirect(url_for('hello'))
+                return redirect(url_for('hello')),200
         else:
-            flash('Login unsuccessful')
+            flash('Login unsuccessful'),401
```
Comment (DiptoChakrabarty) [inline resume/routes.py]: Hey it seems in flask it is preferred to have the syntax in this way as per the flask documentation

redirect(url_for('hello'),code=301) 

returning the status codes in the above method out of the redirect function  seems add bugs to the app as browser requests you to manually redirect to the page .

Also Flask supports status codes 301, 302, 303, 305, and 307 for redirect (as per the documentation you can check), you dont need to give for render_template the status codes . Could you please fix these for the routes.py code

Comment (ShubhamPalriwala) [reply resume/routes.py]: Alright I'll do it !

Comment (DiptoChakrabarty) [reply resume/routes.py]: Also please send the PR's to the development branch as the master is for kicking off heroku deploys , I merge all PRs into master after it is stable in development .

**Comment by ShubhamPalriwala** [comment]
@DiptoChakrabarty Please have a look now !

File: resume/routes.py [hunk 2]
```diff
@@ -370,44 +370,29 @@ def send_reset_pass(user):
...
 @app.route("/reset_account/<token>",methods=["GET","POST"])
 def token_reset(token):
     if current_user.is_authenticated:
-        return redirect(url_for('hello')),200
+        return redirect(url_for('hello'),code=301)
     user_req = user.verify_token(token)
     if user_req is None:
         flash("Token is Invalid or Expired","warning")
-        return redirect(url_for('request_reset')),400
+        return redirect(url_for('request_reset'),code=301)
```
Comment (DiptoChakrabarty) [inline resume/routes.py]: user authenticated and token invalid should have different status codes

Comment (DiptoChakrabarty) [reply resume/routes.py]: can we have different status codes for user authenticated , redirect pages and errors

**Comment by ShubhamPalriwala** [comment]
Should I give the 4xx status codes with the `return(render_template()` if none of the `if` conditions are satisfied ?

**Comment by DiptoChakrabarty** [comment]
Hey @ShubhamPalriwala are you able to run the application , I am still getting the redirect bug in browser .
The previous 302,301 codes were correct , I was asking say for redirect 301 , authentication 302 and errors 307 something similar as other status codes are resulting in errors

**Comment by ShubhamPalriwala** [comment]
@DiptoChakrabarty I've modified the redirect status codes and they should work now for you too !

**Review by DiptoChakrabarty — APPROVED** [review]

**Pull request merged**